build.gradle

```diff
@@ -92,7 +92,7 @@ dependencies {
     implementation("org.apache.logging.log4j:log4j-core:${log4jVersion}")
     implementation("org.opensearch.client:opensearch-rest-high-level-client:${opensearchVersion}")
     implementation("org.opensearch.client:opensearch-rest-client:${opensearchVersion}")
-    implementation("org.opensearch.client:opensearch-java:${opensearchVersion}")
+    implementation("org.opensearch.client:opensearch-java:2.1.0-SNAPSHOT")
```

Comment:
Let's create a variable for the client version too

Reply:
Done